pbiviz.json

```diff
@@ -4,7 +4,7 @@
     "displayName": "Dot Plot",
     "guid": "DotPlot1442374105856",
     "visualClassName": "DotPlot",
-    "version": "0.9.0",
+    "version": "0.9.1",
```

Comment:
Please increase version of the visual to 1.0.1 due to minimal version at Office Store